app/views/posts/index.html.erb

```diff
@@ -1,2 +1,32 @@
-<h1>Posts #index</h1>
-<h2>This is the posts index page</h2>
\ No newline at end of file
+ <div class="main-container">
+ <div class="users-wrapper flex">
+    <div class="img-cont">
+      <%= image_tag @user.photo, :size => "50x50", alt: "user photo" %>
+    </div>
+    <div class="user-details">
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+ </div>
+
+ <% @posts.each do |post| %>
```

Comment:
- [ ] When I visit this page I get this error message in the image below. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/81468675/215783014-d589d05b-cd34-4531-9806-0abb5fb36e18.png)

  In your index action in the posts controller, you need to change `user_id` to `author_id` since that is what you have in the Post table